.circleci/config.yml

```diff
@@ -534,6 +534,34 @@ jobs:
           root: coverage
           paths:
             - codeclimate.sortitions.json
+  blogs:
```

Comment:
This job is not being used, you need to add it to the workflow at the end of the file. You'll also need to modify the `upload-coverage` job to increase the number of the `-p` parameter, and make that job depend on the `blogs` one.